include/mbedtls/arc4.h

```diff
@@ -19,6 +19,11 @@
  *  limitations under the License.
  *
  *  This file is part of mbed TLS (https://tls.mbed.org)
+ *
+ * \warning        ARC4 is considered a weak cipher and its use
+ *                 constitutes a security risk. It is recommended
+ *                 to use a strong cipher instead.
```

Comment:
The problem with these warnings are it begs the question ‘what is a strong cipher’ or ‘strong message digest’? That is a question we want don't want to have to answer in support or in the forums, as users need to choose for themselves what suits them best. (And there's no end of bodies such as NIST or ENISA who'll recommend something).

Therefore I would prefer the wording:


> ARC4 is considered a weak cipher and its use constitutes a security risk. It is recommended alternative ciphers should be considered instead.

Reply:
@sbutcher-arm Thanks for that suggestion, I'll change the wordings accordingly.

Reply:
Done.